ui/packages/ui/src/app/app.tsx

```diff
@@ -0,0 +1,24 @@
+import React  from "react";
+import { Page } from "@patternfly/react-core";
+import {BrowserRouter as Router, Route} from 'react-router-dom';
+import AppHeader from "./appHeader";
+import DashboardPage from "./pages/dashboard/dashboard";
```

Comment:
Hey @mdrillin , for me the landing page was not coming correctly. Updating import as below helped.

import {DashboardPage} from "./pages";

Reply:
Sure, thanks